mmcv/ops/tin_shift.py

```diff
@@ -18,6 +18,12 @@ class TINShiftFunction(Function):
 
     @staticmethod
     def forward(ctx, input, shift):
+        batches_input = input.size(0)
+        batches_shift = shift.size(0)
+        if batches_input != batches_shift:
+            raise ValueError('batches_input should be equal to batches_shift, '
+                             f'but got batches_input={batches_input} \
+                             and batches_shift={batches_shift}.')
```

Comment:
```suggestion
        if input.size(0) != shift.size(0):
            raise ValueError(
                    'The first dim (batch) of `input` and `shift` should be same, '
                    f'but got {input.size(0)} and {shift.size(0)}.')
```

Reply:
Hi @zhouzaida , I refer to `C = input.size(2)` which follow the code, so I use a variable. I will update it.